### DISCUSSION TITLE
`interface` naming convention - Should they always begin with `I`?

### DISCUSSION THREAD

**Discussion by nickyito** [nishiki-tech/nishiki-frontend#77] [General]
Migrated from https://github.com/genesis-tech-tribe/nishiki-frontend/pull/68#discussion_r1421617134
See: https://github.com/orgs/genesis-tech-tribe/discussions/77

<img width="731" alt="image" src="https://github.com/genesis-tech-tribe/nishiki-frontend/assets/99148565/fb1921ba-a536-4d1f-a0d2-0938192ff79a">

**Comment by kotaaaa** [comment]
Let's prioritize consistency of interface naming rule this time, and give it a try. How about it?
@uskayyyyy @kanta1207

**Reply by nickyito** [reply]
You mean names starting with `I`, don't you?

**Reply by nickyito** [reply]
Since this is not a significant matter, so we could just try the naming with `I` as you suggested.

**Reply by kotaaaa** [reply]
Yes, I meant with `I`!

**Comment by nickyito** [comment]
By the way, it seems naming without `I` is a preferred way by officials.

> Do not use `I` as a prefix for interface names.
https://github.com/microsoft/TypeScript/wiki/Coding-guidelines

**Comment by nickyito** [comment]
This discussion has concluded that we will use interface names starting with `I`.
